modules/ROOT/pages/setup.adoc

```diff
@@ -113,10 +114,23 @@ The follow the steps explain how to setup the Payload Storage API:
 . Download the pre-packaged API specification (in RAML) and sample implementation template from Anypoint Exchange.
 Install this API on a Mule runtime that can access the storage solution.**[MISSING INFO - link from Anypoint Exchange for the pre-packaged API and integration app from Manik / Pablo]**
 
-. Make a note of the API address configurations - host, port, path and authentication details.
+. Make a note of the API address configurations (make a note of the following):
+
+** host, port, path connection
+
+** response timeouts
+
+** authentication details: None, API Key, Basic, Client Credentials, OAuth Token
+
+** Certificate (.PEM format) and connection type
+
+[xref] to HTTPS Send Configuration for complete details?
+
 You'll need these details while associating Anypoint Partner Manager to your Payload Storage API. +
 
-NOTE: Your payload storage API must be accessible over the internet via HTTPS. Ensure that the appropriate authentication mechanisms have been implemented to make this happen.
+NOTE: Your payload storage API must be accessible over the internet via HTTPS.
+Ensure that the appropriate authentication mechanisms have been implemented to make this happen.
+For example, if you are using AS2, you'll need to understand from your administrator what the hostname is so you can tell your partner where to send the AS2 transmission.
```

Comment:
This doesn't look related to the previous statement

Reply:
REmoved per Pablo's comments on B2BSOULT-2387